libs/features/pages/contact-page/src/lib/components/contact-page-form-section/contact-page-form-section.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { FormControl,FormGroup } from '@angular/forms';
+@Component({
+  selector: 'elewa-website-contact-page-form-section',
+  templateUrl: './contact-page-form-section.component.html',
+  styleUrls: ['./contact-page-form-section.component.scss'],
+})
+export class ContactPageFormSectionComponent {
+  buttonData = {
+    text: 'Send message',
+    /** base colors on button */
+    color: '#ffffff',
+    bgColor: '#292A50',
+    iconColor: '#000000',
+    borderColor: '#000000',
+    iconBackgroundColor: '#ffffff',
+
+    /** hover effect styles on the button  */
+    hoverColor: '#000000',
+    hoverBgColor: '#ffffff',
+    hoverIconColor: '#000000',
+    hoverBorderColor: '#00000',
+    hoverIconBackgroundColor: '#000000',
+  };
```

Comment:
For the sake of keeping the component lean and ensuring readability and maintainability, this can be moved to its own Typescript file in the data module